layers/tests/test_decorators.py

```diff
@@ -27,27 +27,27 @@ def test_exclude_from_layers_basic(self):
         response = self.client.get(url)
         result = response.content
         self.assertEqual(response.status_code, 200)
-        self.failUnless("This is a normal view" in result)
+        self.failUnless(str.encode("This is a normal view") in result)
 
         url = reverse("web-only-view")
         response = self.client.get(url)
         result = response.content
         self.assertEqual(response.status_code, 200)
-        self.failUnless("is not available for your device" in result)
+        self.failUnless(str.encode("is not available for your device") in result)
 
     @override_settings(LAYERS=WEB_LAYERS_)
     def test_exclude_from_layers_web(self):
         url = reverse("normal-view")
         response = self.client.get(url)
         result = response.content
```

Comment:
I think it may be cleaner to do `response.content.decode()` rather than encode the stirng later.